bin/admin/bootstrap-gce.sh

```diff
@@ -31,4 +31,8 @@ echo $enablestring >> $acctfile
 echo "install conda with Python 3 and 2 ..."
 $HOME/opt/gce/bin/admin/install-conda.sh
 
+# get ready to use SOLVCON
+echo "building SOLVCON..."
+$HOME/opt/gce/bin/admin/install-solvcon.sh $acctfile
```

Comment:
I agree with you. The design of a push button command sounds reasonable to me.

I install SOLVCON at the bootstrapping stage because I suppose all nodes will be used for the dealer (server) and the workers(clients) of SOLVCON. However these scenarios may not need SOLVCON source code to be pre-installed:

* another parallel infrastructure suite deployed for SOLVCON
* a frontend to visualize outputs of SOLVCON

I will remove the hard-coded part to install SOLVCON at the bootstrapping stage.